modules/telco-ran-machine-configuration.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-ran-machine-configuration_{context}"]
+= Machine configuration
+
+New in this release::
+* Set `rcu_normal` after node recovery
+
+Limits and requirements::
+* The CRI-O wipe disable `MachineConfig` assumes that images on disk are static other than during scheduled maintenance in defined maintenance windows.
+To ensure the images are static, do not set the Pod `imagePullPolicy` field to `Always`.
++
+.Machine configuration options
+[cols=2*, width="90%", options="header"]
+|====
+|Feature
+|Description
+
+|Container runtime
+|Sets the container runtime to `crun` for all node roles.
+
+|kubelet config and container mount hiding
+|Reduces the frequency of kubelet housekeeping and eviction monitoring to reduce CPU usage.
+Create a container mount namespace, visible to kubelet and CRI-O, to reduce system mount scanning resource usage.
+
+|Recovery optimization systemd service
+|Assigns additional CPUs to systemd services during reboot recovery
```

Comment:
This machine config was removed from the table above but missed here (and in source).